wiki/en/en-Privacy-Statement.md

```diff
@@ -22,9 +22,7 @@ When you connect to a public server, your profile is also available to third par
 
 When you connect to a public or private server, the server operator can see your IP address while you are connected.  If the server operator has enabled logging (which is off by default) your IP address will also be logged and stored in the server's log file.
 
-As a server operator, when you register a public server with a Central Server, your IP address is sent to CloudFlare (1.1.1.1) in order to identify your public IP address. The IP addresses of all public servers registered with the Central Server can also be seen by third parties for informational or other purposes (for example [here](http://explorer.jamulus.io/)). Your public IP address is otherwise not logged or stored by Jamulus, but may be stored or processed by third parties.
-
-_Users concerned by the use of CloudFlare's network can [re-compile](Compiling) the Jamulus source code to use an alternative for WELL_KNOWN_HOST/ WELL_KNOWN_PORT in [global.h](https://github.com/corrados/jamulus/blob/master/src/global.h#L111)_
+The IP addresses of all public servers registered with the Central Server can also be seen by third parties for informational or other purposes (for example [here](http://explorer.jamulus.io/)). Your public IP address is otherwise not logged or stored by Jamulus, but may be stored or processed by third parties.
```

Comment:
Or, moving away from "public" and "Central Server"...
`The IP address of any server registering with a directory server can also be seen by third parties accessing the same directory server for...`

Reply:
I agree, but as this PR skips the translation process by doing the deletion for all languges, so I don't want to make any text changes here. Guess this should be a new ticket to re-work the Directory/Central server thing for the docs next release.

Reply:
Yes - at the risk of being a bit bureaucratic, the central server vocab thing is a separate piece of work. https://github.com/jamulussoftware/jamuluswebsite/issues/330